cockatrice/src/carddatabase.cpp

```diff
@@ -830,4 +900,26 @@ bool CardDatabase::saveCustomTokensToFile()
     xml.writeEndDocument();
 
     return true;
+}
+
+CardRelation::CardRelation(const QString &_name,
+		bool _doesAttach,
```

Comment:
I'm not using all tabs, it's just that one line. A hiccup with my IDE. :)

There are actually a scattering of tabs elsewhere in the code as well.